src/_posts/security/procedures/2000-01-01-external-testing.md

```diff
@@ -47,4 +47,4 @@ For example:
 4. If there are any items in the report that would be of interest to Scalingo,
    please send us the information so that we can act accordingly
 
-{% include security_testing_warning.md %}
\ No newline at end of file
+{% include security_testing_warning.md %}
```

Comment:
It's really not clear on Github interface, but this is adding a end of file new line